pkg/components/velero/azure/azure.go

```diff
@@ -15,6 +15,10 @@
 package azure
```

Comment:
There is typo in commit message: `tempplate.go`.

Also, commit message is outdated, `Values()` method do not return values template, but provider-specific values.